db/migrations/02_create_users_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE users(id SERIAL PRIMARY KEY, name VARCHAR (50), username VARCHAR(20), email VARCHAR(50), password VARCHAR(20));
```

Comment:
duplicates db/test/01_create_peeps_test_table.sql?

Reply:
It does. I initially was setting up a test table with test naming conventions then realised that it won't work so scrapped that then didn't delete the file!